tests/conftest.py

```diff
@@ -51,19 +51,16 @@ def _app_factory(config=None):
         SECURITY_PASSWORD_SALT="CHANGE_ME_ALSO",
         SECURITY_CONFIRM_EMAIL_WITHIN="2 seconds",
         SECURITY_RESET_PASSWORD_WITHIN="2 seconds",
+        ACCOUNTS_SESSION_STORE_FACTORY=(
```

Comment:
Isn't that the default already from `config.py`?

Reply:
👍 updated